frontend/src/queries/nodes/DataTable/ColumnConfigurator/ColumnConfigurator.tsx

```diff
@@ -91,35 +92,38 @@ function ColumnConfiguratorModal(): JSX.Element {
             <SortableDragIcon />
         </span>
     ))
-    const SelectedColumn = ({ column }: { column: string }): JSX.Element => {
-        let columnType: PropertyFilterType | null = null
-        let columnKey = column
-        if (column.startsWith('person.properties.')) {
-            columnType = PropertyFilterType.Person
-            columnKey = column.substring(18)
-        }
-        if (column.startsWith('properties.')) {
-            columnType = PropertyFilterType.Event
-            columnKey = column.substring(11)
-        }
-
+    const SelectedColumn = (props: { column: string; dataIndex: number }): JSX.Element => {
+        const { column, dataIndex } = props
         return (
             <div className={clsx(['SelectedColumn', 'selected'])} style={{ height: rowItemHeight }}>
                 <DragHandle />
-                {columnType && <PropertyFilterIcon type={columnType} />}
-                <PropertyKeyInfo className="ml-1" value={columnKey} />
-                <div className="text-right flex-1">
-                    <Tooltip title={'Remove'}>
-                        <LemonButton
-                            onClick={() => unselectColumn(column)}
-                            status="stealth"
-                            size="small"
-                            className="ml-auto"
-                        >
-                            <IconClose data-attr="column-display-item-remove-icon" style={{ color: 'var(--danger)' }} />
-                        </LemonButton>
-                    </Tooltip>
-                </div>
+                <code>{column}</code>
+                <div className="flex-1" />
+                <Tooltip title="Edit">
+                    <LemonButton
+                        onClick={() => {
+                            const newColumn = window.prompt('Edit column', column)
```

Comment:
Old-school

Reply:
yeah, this entire thing needs to change. I went for the quickest thing I could find